### PR TITLE
Add TARDIS Bot comments from TARDIS Repo

### PR DESCRIPTION
**Pull request by smokestacklightnin**
### :pencil: Description

**Type:** | :roller_coaster: `infrastructure`

This includes TARDIS bot actions to leave comments for success or failure of the build-docs workflow.

### :pushpin: Resources

[Original code](https://github.com/tardis-sn/tardis/blob/334c6fe7324b78599f74f1e266be0de0e3abbf79/.github/workflows/build-docs.yml#L134-L178)

### :vertical_traffic_light: Testing

How did you test these changes?

- [X] Not sure how to test this on my own branch

### :ballot_box_with_check: Checklist

- [x] I requested two reviewers for this pull request
- [ ] I updated the documentation according to my changes
- [ ] I built the documentation by applying the `build_docs` label

### CODE REVIEW

**Comment by jvshields** [comment]
Can you describe what you've done to test if this workflow runs correctly? Does it just need to be merged before we can see if it works or are there any other things we could do to test it?

**Comment by smokestacklightnin** [comment]
> Can you describe what you've done to test if this workflow runs correctly? Does it just need to be merged before we can see if it works or are there any other things we could do to test it?

I will see what I can do to test it. Apologies for not doing that beforehand.

**Comment by wkerzendorf** [comment]
I'm not quite sure why we need this? Could you add a bit more description?

**Comment by smokestacklightnin** [comment]
> I'm not quite sure why we need this? Could you add a bit more description?

I don't have an opinion about whether it's necessary or not. I included it to match the [TARDIS workflow](https://github.com/tardis-sn/tardis/blob/master/.github/workflows/build-docs.yml).

File: .github/workflows/build-docs.yml [hunk 1]
```diff
@@ -91,3 +91,49 @@ jobs:
...
+            
+            [**Click here**](${{ env.URL }}) to see your results.
+        env:
+          URL: https://${{ github.repository_owner }}.github.io/${{ github.event.repository.name }}/pull/${{ github.event.number }}/index.html
+        if: success() && github.event_name == 'pull_request_target'
+
+      - name: Post comment (failure)
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          token:  ${{ secrets.GITHUB_TOKEN }}
```
Comment (atharva-2001) [inline .github/workflows/build-docs.yml:125]: tardis bot token, right?
Is this saved as a secret?

**Comment by smokestacklightnin** [comment]
> I'm not quite sure why we need this? Could you add a bit more description?

Something very useful that this does is [have the tardis bot comment with a URL to the version of the docs on the branch receiving the change](https://github.com/tardis-sn/stardis/pull/80/files#diff-4fdc48ab490b670c90974725b8e3d399b24c03d6d09563acf8ea87d4113930fbR119)

**Review by jvshields — DISMISSED** [review]

File: .github/workflows/build-docs.yml (hunk 1 above)
Comment (smokestacklightnin) [reply .github/workflows/build-docs.yml:125]: This line works in other workflows, although I don't know the direct answer to your question off the top of my head.

Comment (atharva-2001) [reply .github/workflows/build-docs.yml:125]: I think we should use the `BOT_TOKEN` here. This also needs to be added in the repo settings.

Comment (smokestacklightnin) [reply .github/workflows/build-docs.yml]: > I think we should use the `BOT_TOKEN` here. This also needs to be added in the repo settings.

Is [this](https://github.com/tardis-sn/stardis/pull/80/commits/2933599237dad1c90dbaf4b7ca3c8949ef90c418) what you had in mind?

**Review by jvshields — APPROVED** [review]

File: .github/workflows/build-docs.yml (hunk 1 above)
Comment (atharva-2001) [reply .github/workflows/build-docs.yml]: yes! thank you.

**Review by atharva-2001 — APPROVED** [review]

**Pull request merged**